src/Extractor/SnowflakeConnectionFactory.php

```diff
@@ -145,7 +145,7 @@ protected function getUserDefaultWarehouse($connection, DatabaseConfig $database
             $this->quoteIdentifier($databaseConfig->getUsername())
         ));
 
-        while ($item = odbc_fetch_array($stmt)) {
+        while ($stmt && $item = odbc_fetch_array($stmt)) {
```

Comment:
`composer update`. updated phpstan which then complained that `$stmt` here was `false|resource` so I put a check that it wasn't false.

Reply:
I reverted the stan update:)